proto/stride/stakeibc/host_zone.proto

```diff
@@ -7,6 +7,19 @@ import "cosmos_proto/cosmos.proto";
 
 option go_package = "github.com/Stride-Labs/stride/v19/x/stakeibc/types";
 
+// CommunityPoolRebate stores the size of the community pool liquid stake
+// and the rebate percentage
+message CommunityPoolRebate {
+  string rebate_percentage = 1 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+  string liquid_stake_amount = 2 [
```

Comment:
nit: I think `liquid_staked_amount` (past tense) would be more clear. I found the present tense confusing [here](https://github.com/Stride-Labs/stride/pull/1136/files#diff-8850ef830f8cbe0e6ca0be787be1036bc793215c526fb43491a812ca4f0bb079R108), made me think we were referring to tokens in the process of being liquid staked.

Reply:
totally agree! going to leave this comment open and address it once everything's merged into this so I don't have to deal with a bunch of annoying merge conflicts

Reply:
@riley-stride going to handle this in the PR that changes the % calculations to use stDYDX instead of DYDX.

Do you have a name preference? Maybe `liquid_staked_sttoken_amount` ?

Reply:
`liquid_staked_sttoken_amount` sgtm!

Reply:
sweet - going to mark this resolved since it will be in the other PR